install.rb

```diff
@@ -36,8 +36,9 @@
 require 'optparse'
 require 'ostruct'
 
-PREREQS = %w{openssl facter cgi}
-MIN_FACTER_VERSION = 1.5
+IS_WINDOWS = RUBY_PLATFORM.match?(/(mswin|mingw)/)
```

Comment:
Will not rubocop if we enable it one day complain that we should freeze constants?  It is probably something we will do today, so probably not a big deal for now.

Reply:
There's a lot more to rubocop in this file ;)